configure.ac

```diff
@@ -93,7 +93,7 @@ if test "x$build_tests" = "xyes"; then
                       [cunit],
                       [have_cunit=yes
                        AC_DEFINE(HAVE_LIBCUNIT, [1], [Define if the system has libcunit])]
-                       AC_DEFINE(TEST_INCLUDE, [1], [Define to include test wrappers for static functions])],
+                       AC_DEFINE(TEST_INCLUDE, [1], [Define to include test wrappers for static functions]),
```

Comment:
I think that the `]` is correct in this line. The spurious `]` is in the previous line.

Reply:
Corrected and pushed.